examples/add-kafkametrics-receiver/rendered_manifests/configmap-agent.yaml

```diff
@@ -95,6 +95,41 @@ data:
         - sources:
           - from: resource_attribute
             name: host.name
+      k8sattributes/metrics:
+        extract:
+          annotations:
+          - from: pod
+            key: splunk.com/sourcetype
+          - from: namespace
+            key: splunk.com/metricsIndex
+            tag_name: com.splunk.index
+          - from: pod
+            key: splunk.com/metricsIndex
+            tag_name: com.splunk.index
+          metadata:
+          - k8s.node.name
+          - k8s.namespace.name
```

Comment:
Why are these two needed? All the k8s metrics should go with these attributes already.

Reply:
I used to be accidentally required due to a bug which was fixed recently https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/28838. Make sure you use +0.90.0 image

Reply:
Yeah, just checked it out with a new image and it seems to be working fine. Thanks :)